test/system/090-events.bats

```diff
@@ -25,3 +25,35 @@ load helpers
     run_podman events --filter type=container --filter container=$cname --filter event=start --stream=false
     is "$output" "$expect" "filtering just by label"
 }
+
+@test "image events" {
+    # `remove events` seem to be broken...
+    skip_if_remote "FIXME: -remote is missing many events"
+    pushedDir=$PODMAN_TMPDIR/dir
+    mkdir -p $pushedDir
+
+    tarball=$PODMAN_TMPDIR/ball.tar
+
+    run_podman image inspect --format "{{.ID}}" $IMAGE
+    imageID="$output"
+
+    tag=registry.com/$(random_string 10 | tr A-Z a-z)
+
+    run_podman push $IMAGE dir:$pushedDir
+    run_podman save $IMAGE -o $tarball
+    run_podman load -i $tarball
+    run_podman pull docker-archive:$tarball
+    run_podman tag $IMAGE $tag
+    run_podman untag $IMAGE $tag
+    run_podman tag $IMAGE $tag
+    run_podman rmi $tag
+
+    run_podman events --stream=false --filter type=image
```

Comment:
I would urge you please to add `--since`:
```bash
    t0=$(date --iso-8601=seconds)
    run_podman etc etc etc
    ...
    run_podman events --stream=false --filter type=image --since=$t0
```
Without that, the `events` command takes a long time on my laptop, which has only been up for a week or so; it would be pretty bad on a system with a multi-week uptime.

Reply:
Lovely, thanks!